ros2bag/ros2bag/verb/info.py

```diff
@@ -22,7 +22,15 @@ class InfoVerb(VerbExtension):
 
     def add_arguments(self, parser, cli_name):  # noqa: D102
         add_standard_reader_args(parser)
+        parser.add_argument(
+            '-t', '--topic_name', action='store_true',
+            help='Only display topic names.'
+        )
 
     def main(self, *, args):  # noqa: D102
         m = Info().read_metadata(args.bag_path, args.storage)
-        print(m)
+        if args.quiet:
```

Comment:
Forgot to rename to `if topic_name:` ?
It would be nice to add some simple sanity unit test similar to test for playback
https://github.com/ros2/rosbag2/blob/c7c7954d4d9944c160d7b3d716d1cb95d34e37e4/ros2bag/test/test_play_qos_profiles.py#L62-L72
Perhaps we can use empty bag from resources folder for reading info.

Reply:
@MichaelOrlov 
> Forgot to rename to if topic_name: ?

Yes :cry: Fixed at 9014a16

> It would be nice to add some simple sanity unit test similar to test for playback

Thanks for a good comment. I've added tests for info verb. a1b450d